src/platforms/java/common/configuration/index.mdx

```diff
@@ -133,6 +133,17 @@ To set the server name that will be sent with each event, use the `servername` o
 servername=host1
 ```
 
+### Timeout (Advanced)
+
+A timeout is set to avoid blocking Sentry threads because establishing a connection is taking too long.
```

Comment:
I don't know what a "Sentry thread" is here.

Reply:
I removed it. Agreed that it can be confusing.